src/core/menhir_parser.mly

```diff
@@ -250,7 +250,10 @@ line:
         | Some(_) -> ms
         | None -> make_pos Lexing.(dummy_pos, dummy_pos) (P_prop(Const)) :: ms
       in
-      make_pos $loc (P_symbol(ms, make_pos $loc(s) s, ps, a))
+      let t =
+        P_symbol(ms, make_pos $loc(s) s, ps, a, None)
```

Comment:
coding style: can't this let and the following fit on one line?

Reply:
fixed in 84e565a4